ui/app/AppLayouts/Profile/Sections/SyncHistoryModal.qml

```diff
@@ -0,0 +1,70 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtQuick.Layouts 1.13
+import "../../../../imports"
+import "../../../../shared"
+import "../../../../shared/status"
+
+ModalPopup {
+    id: popup
+    title: qsTr("Sync history for")
+    
+    Column {
+        id: column
+        anchors.top: parent.top
+        anchors.bottom: parent.bottom
+        anchors.left: parent.left
+        anchors.right: parent.right
```

Comment:
I think `anchors.fill: parent` does the job here. If the margins below don't work, then instead you can still do `height: parent.height` to save a line 😁